seed/views/00602_ldu_court_reporter_overview.js

```diff
@@ -1,24 +1,35 @@
 exports.seed = function (knex, promise) {
-  var sql = `CREATE VIEW app.ldu_court_reporter_overview
+  var view = `CREATE VIEW app.ldu_court_reporter_overview
     WITH SCHEMABINDING
     AS
     SELECT
-      MAX(l.id) AS id
-      , MAX(t.description) AS name
-      , MAX(tv.id) AS link_id      
-      , MAX(tv.grade_code) AS grade_code
-      , SUM(tv.contracted_hours) AS contracted_hours
-      , SUM(tv.reduction_hours) AS reduction_hours
-      , SUM(tv.total_sdrs) AS total_sdrs
-      , SUM(tv.total_fdrs) AS total_fdrs
-      , SUM(tv.total_oral_reports) AS total_oral_reports
-    FROM app.team_court_reporter_overview tv
-      JOIN app.team t ON tv.id = t.id
-      JOIN app.ldu l ON l.id = t.ldu_id
-    GROUP BY tv.id;`
+        l.id
+      , t.description AS name
```

Comment:
shouldn't it be returned as description? it will make it consistent between the attribute and what is returned.

Reply:
This allows our web app access code to be simpler as the majority of views have a `name` property.

Reply:
I will go with your comment. Not ideal but its minor thing, so happy with the comment.